views/includes/inputs/presenter-input.html

```diff
@@ -0,0 +1,40 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    {% include "includes/list/errors.html" %}
+
+    <h1 class="govuk-heading-xl">{{ title }}</h1>
+    <p class="govuk-caption-m govuk-!-margin-bottom-4">We’ll use this if we need more information about the {{ contentExtension }}.</p>
```

Comment:
can we think of a better name for contentExtension. Does "application" not work for both registration and update as they are both a sort of application (to register or update)?

Reply:
The content that changes is "application" for Registration and "update" for the Update journey, so we might need another name than application I think....

Reply:
submission? with values application or "update application"?

Reply:
will replace contentExtension with submission, its update as confirmed by David Wright...